### PR TITLE
Heroku Deploy

### PR DESCRIPTION
**Pull request by hoadam**
## Type of change
- [ ] New feature
- [ ] Bug Fix
- [ ] Test
- [ ] Refactor/Style

## Description
Relevant User Story(s):
*Brief summary of the change(s) included in this pull request*

This description closes #

## Check the correct boxes
- [ ] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes
- [ ] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed

*Please describe what in the world happened if tests are broken*

## Checklist:
- [ ] My code has no unused/commented out code
- [ ] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code

### CODE REVIEW

File: Procfile.dev [hunk 1]
```diff
@@ -1,3 +1,4 @@
 web: bin/rails server
 
 css: bin/rails tailwindcss:watch
+js: yarn build --watch
```
Comment (yainporter) [inline Procfile.dev:4]: I see you had to build with yarn!

File: bin/dev [hunk 2]
```diff
@@ -6,7 +6,7 @@ if ! gem list foreman -i --silent; then
 fi
 
 # Default to port 3000 if not specified
-export PORT="${PORT:-5000}"
+export PORT="${PORT:-3000}"
```
Comment (yainporter) [inline bin/dev:10]: Wait, why is the port changed?

Comment (hoadam) [reply bin/dev:10]: oh shoot, I will change back to 5000

File: bin/dev [hunk 3]
```diff
@@ -6,7 +6,7 @@ if ! gem list foreman -i --silent; then
...
```
Comment (hoadam) [inline bin/dev:9]:
```suggestion
export PORT="${PORT:-5000}"
```

File: bin/dev (hunk 2 above)
Comment (yainporter) [reply bin/dev]: Thanks!

**Review by yainporter — APPROVED** [review]
Thanks for getting it deployed!

**Review by Selena730 — APPROVED** [review]
Thank you Hoa!

**Pull request merged**